providers/mailjet/src/lib/mailjet.provider.ts

```diff
@@ -44,6 +44,11 @@ export class MailjetEmailProvider implements IEmailProvider {
           Subject: options.subject,
           TextPart: options.text,
           HTMLPart: options.html,
+          Attachments: options.attachments.map((attachment) => ({
```

Comment:
```suggestion
          Attachments: options.attachments?.map((attachment) => ({
```

In case attachments are not passed